publish.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+JOB_TYPE=${JOB_TYPE:-local}
+REPO_OWNER=${REPO_OWNER:-openshift}
+REPO_NAME=${REPO_NAME:-dynamic-plugin-sdk}
+PULL_NUMBER=${PULL_NUMBER:-0}
+NPM_TOKEN=${NPM_TOKEN:-notoken}
+FORCE_PUBLISH=${FORCE_PUBLISH:-false}
+
+LABEL_PUBLISH="publish-sdk"
+PACKAGE_GLOB=(./packages/lib-*)
+NPM_REGISTRY="https://registry.npmjs.org/"
+NPM_REGISTRY_AUTH_KEY="//registry.npmjs.org/:_authToken"
+LABELS=$(curl -s \
+  -H "Accept: application/vnd.github.v3+json" \
+  "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/pulls/${PULL_NUMBER}" \
+  | jq '.labels? | map(.name)?')
+HAS_PUBLISH=$(printf '%s\n' "${LABELS}" | jq 'index("'${LABEL_PUBLISH}'")')
+
+if [[ ( "${JOB_TYPE}" != "local" && "${HAS_PUBLISH}" != "null" ) || "${FORCE_PUBLISH}" == "true" ]]; then
+  echo 'publish npm packages'
+  if [[ "${JOB_TYPE}" != "postsubmit" && "${FORCE_PUBLISH}" != "true" ]]; then
+    echo 'non postsubmit job detected - performing dry run'
+    DRY_RUN="--dry-run"
```

Comment:
Instead of `DRY_RUN`, I'd suggest using `PUBLISH_ARGS` initialized like
```sh
PUBLISH_ARGS=""
```
above the whole `if` block, and update it here like so:
```sh
PUBLISH_ARGS="${PUBLISH_ARGS} --dry-run"
```